src/Map.jsx

```diff
@@ -74,7 +82,40 @@ class Map extends Component {
       { force: true },
     );
   }
+  getStopInfo(route, info) {
+    let sInfo = {};
+    if (!this.state.stopInfo.isFirstStpChosen()) {
```

Comment:
Lots of if else's here, maybe we can clean up

Reply:
I think we could make it simpler. We have a list of selected stops. When adding a second stop to the list, we check whether the first stop on the list is equal to it. If yes, we add it; if not we don't. When adding a third stop to the list, we kick the previous 2 out and add it.

Also, could the selectedStops list be a list of strings representing the stopID? I'm fine with adding a map for accessing a stop (what we get from the backend) via its ID to the state.

Reply:
@EddyIonescu  I like it 👍